### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by jessikamorais**
- [DEMO LINK](https://jessikamorais.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://jessikamorais.github.io/layout_search-bar-airbnb/report/html_report/)

- [ ] Icon implemented using background-image CSS property
- [ ] Inputs are written inside of 'form' tag with correctly passed attributes
- [ ] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [ ] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/styles/style.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  border-radius: 4px;
+
+}
+
+.second_search--style{
+  padding-bottom: 20px;
+  background-color:#2F4F4F;
+  border:solid 1px;
+  border-radius:15px;
+  width:300px;
```
Comment (joaorpereira) [inline src/styles/style.css:28]: the search input should have full width, to accomplish that we need to set width to 100%

File: src/styles/style.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  margin-bottom: 50px;
+  border:solid 1px #E1E7ED;
+  border-radius: 4px;
+
+}
+
+.second_search--style{
+  padding-bottom: 20px;
+  background-color:#2F4F4F;
+  border:solid 1px;
```
Comment (joaorpereira) [inline src/styles/style.css:26]: it's missing the border color

File: src/index.html [hunk 3]
```diff
@@ -13,20 +14,32 @@
     <title>Document</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="./styles/style.css"
```
Comment (joaorpereira) [inline src/index.html:17]: @jessikamorais, you should not change the path of the styles.css file. The PR verification has pre-files selected to verify whether the code you are doing is correct or not. As you created a new file styled under the/styles folder, the verification is not working. Please move this code back to `href="style.css"` and move your whole CSS code inside that file to make code verification work properly.

File: src/style.css [hunk 4]
```diff
@@ -1 +0,0 @@
-/* add styles here */
```
Comment (joaorpereira) [inline src/style.css:1]: @jessikamorais, you should add back this file and use this file to add your css code.

**Comment by joaorpereira** [comment]
@jessikamorais these are the requirements of the task and they are missing in your solution:
- [ ] use images from [src/images](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/src/images)
- [ ] there must be two search bars
- [ ] search bar must stretch the full width
- [ ] distance between two search bars must be 20px
- [ ] the big search bar must have top indent 20px
- [ ] follow styles from the mock
- [ ] default font-weight must be 300
- [ ] a search bar has 3 state default, hover and focus
- [ ] don't use JavaScript
- [ ] use @font-face for fonts
- [ ] add attribute data-qa="big" for big search form, and data-qa="small" for small
- [ ] add attribute data-qa="keypress" to input in big search form

File: src/index.html [hunk 5]
```diff
@@ -13,20 +14,32 @@
...
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search_bars">
```
Comment (joaorpereira) [inline src/index.html:24]: wrap all elements inside a `form` tag

**Review by joaorpereira — CHANGES_REQUESTED** [review]
Pending requirements and build error

File: src/styles/style.css [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+.search_bars {
+  padding: 0 41px;
+  display: inline-block;
+  box-sizing: border-box;
+}
+
+.first_search--style{
+  width: 900px;
```
[the remark this answers is not in the document]
Comment (jessikamorais) [reply src/styles/style.css:14]: I'm sorry, I did the PR before finishing the activity. But thanks for the tips, I'll fix it.

**Comment by jessikamorais** [comment]
Code not finished yet

**Comment by joaorpereira** [comment]
@jessikamorais I believe this is missing: 

`Inputs are written inside of 'form' tag with correctly passed attributes`

File: .prettierignore [hunk 7]
```diff
@@ -1,3 +0,0 @@
-/node_modules
```
Comment (joaorpereira) [inline .prettierignore:1]: add this file back

File: .prettierrc [hunk 8]
```diff
@@ -1,20 +0,0 @@
...
-  "singleQuote": true,
-  "trailingComma": "all",
-  "printWidth": 80,
-  "overrides": [
-    {
-      "files": "*.html",
-      "options": {
-        "parser": "html",
-        "insertPragma": false,
-        "endOfLine": "auto",
```
Comment (joaorpereira) [inline .prettierrc:14]: add this file back

File: src/index.html [hunk 9]
```diff
@@ -16,17 +17,32 @@
...
   </head>
+  <!-- #endregion -->
+
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div
```
Comment (joaorpereira) [inline src/index.html:23]: wrap input element inside form component like:

```
     <form
        class="inputs__form"
        action="post"

        data-qa="keypress"
      >
        <input
          class="inputs__bigInput"
          type="text"
          data-qa="big"
          placeholder="Try “Los Angeles“"
        />
      </form>
```

File: src/index.html [hunk 10]
```diff
@@ -16,17 +17,32 @@
...
+        placeholder="Try “Los Angeles“"
+      />
+    </div>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div
```
Comment (joaorpereira) [inline src/index.html:36]: same here

File: src/style.css [hunk 11]
```diff
@@ -1 +1,86 @@
 /* add styles here */
+
+@font-face {
```
Comment (joaorpereira) [inline src/style.css:3]:
```
@font-face {
  font-family: 'Avenir';
  src: url(fonts/Avenir.ttc);
  font-weight: 300;
}

@font-face {
  font-family: 'Avenir';
  src: url(fonts/Avenir-Heavy.ttf);
  font-weight: 900;
}

@font-face {
  font-family: 'Avenir';
  src: url(fonts/Avenir-Book.ttf);
  font-weight: 400;
}
```

File: src/style.css [hunk 12]
```diff
@@ -1 +1,86 @@
...
+  font-family: AvenirFonte;
+  src:
+    url('fonts/Avenir-Book.ttf') format('truetype'),
+    url('fonts/Avenir-Heavy.ttf') format('truetype'),
+    url('fonts/Avenir.ttc') format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
```
Comment (joaorpereira) [inline src/style.css:12]: .html {
  font-family: 'Avenir';
  font-weight: 300;
}

File: src/style.css [hunk 13]
```diff
@@ -1 +1,86 @@
...
+body {
+  margin: 0;
+  font-family: AvenirFonte, Arial, sans-serif;
```
Comment (joaorpereira) [inline src/style.css:15]: remove this line

**Review by joaorpereira — CHANGES_REQUESTED** [review]

**Pull request closed without merging**